src/index.ts

```diff
@@ -6,12 +6,28 @@ import swaggerSpec from "./docs/swaggerconfig";
 import "reflect-metadata";
 import { connect } from './database';
 import route from './router';
+import fs from "fs"
+import path from "path"
 
 const app: Application = express();
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
+
+
+let logStream = fs.createWriteStream(path.join(__dirname,'output.log'), {
+  flags: 'a'
+})
+
+morgan.token('type', function (req,res){
+return req.headers['content-type']
+})
+
+app.use(cors());
+app.use(morgan(":method :url :status :res[content-lenght] - :response-time ms :date[web] :type",{ stream: logStream}));
```

Comment:
You can only do `combined` than adding `":method :url :status :res[content-lenght] - :response-time ms :date[web] :type"`